src/components/ModalComponent.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
Could we build this modal using material UI's modal as a base?

Reply:
Thanks, @kkyeboah 🙂  I used material-UI modal this time.